app/views/static_pages/help.html.erb

```diff
@@ -0,0 +1,8 @@
+<% provide :title, "Help" %>
```

Comment:
I18n cho file này và toàn bộ file html nhé
-> ko chỉ làm mẫu 1 file đâu

ngoài ra, thay vì dùng `a` thì dùng `link_to` nhé